framework/src/play/mvc/Scope.java

```diff
@@ -37,6 +34,19 @@
     public static final boolean SESSION_SEND_ONLY_IF_CHANGED = Play.configuration
             .getProperty("application.session.sendOnlyIfChanged", "false").toLowerCase().equals("true");
 
+    public static SessionStore sessionStore = createSessionStore();
+
+    private static SessionStore createSessionStore() {
+        String sessionStoreParameter = Play.configuration.getProperty("application.session.store", "cookie");
```

Comment:
Would be better if we just have application.session.storeClass which takes in the class instead of relying on some naming convention that has a "SessionStore" suffix.